### PR TITLE
linters: enable stylecheck

### PR DESCRIPTION
**Pull request by melekes**
https://github.com/tendermint/tendermint/pull/3961

This PR repairs linter errors seen when running the following command:
golangci-lint run --no-config --disable-all=true --enable=stylecheck

Contributes to #3262

* [x] Referenced an issue explaining the need for the change
* [ ] ~~Updated all relevant documentation in docs~~
* [ ] ~~Updated all code comments where relevant~~
* [ ] ~~Wrote tests~~
* [x] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: p2p/conn/secret_connection.go [hunk 1]
```diff
@@ -296,21 +296,21 @@ func shareEphPubKey(conn io.ReadWriter, locEphPub *[32]byte) (remEphPub *[32]byt
 
 	// Send our pubkey and receive theirs in tandem.
 	var trs, _ = cmn.Parallel(
-		func(_ int) (val interface{}, err error, abort bool) {
+		func(_ int) (val interface{}, abort bool, err error) {
 			var _, err1 = cdc.MarshalBinaryLengthPrefixedWriter(conn, locEphPub)
 			if err1 != nil {
-				return nil, err1, true // abort
+				return nil, false, err1 // abort
```
Comment (melekes) [inline p2p/conn/secret_connection.go]: !!!!!!!!!!!!!!!!!!!

Comment (melekes) [reply p2p/conn/secret_connection.go]: TODO: create an issue because tests weren't failing

Comment (melekes) [reply p2p/conn/secret_connection.go]: https://github.com/tendermint/tendermint/issues/4154

File: CHANGELOG_PENDING.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ program](https://hackerone.com/tendermint).
 - Go API
   - [libs/pubsub] [\#4070](https://github.com/tendermint/tendermint/pull/4070) `Query#(Matches|Conditions)` returns an error.
   - [rpc/client] \#3471 `Validators` now requires two more args: `page` and `perPage`
+  - [libs/common] \#3262 Make error the last parameter of `Task` (@PSalant726)
```
Comment (ebuchman) [inline CHANGELOG_PENDING.md]: also cryptoAmino rename.

Also I thought I saw some `XYZError` get renamed to `ErrXYZ` but I can't find it now and not sure it was even in a package covered by our versioning

Comment (melekes) [reply CHANGELOG_PENDING.md]: > also cryptoAmino rename.

how's this breaking? I am not even sure it's worth mentioning in the changelog

Comment (ebuchman) [reply CHANGELOG_PENDING.md]: isn't it a change in package name?

Comment (melekes) [reply CHANGELOG_PENDING.md]: imo it's only breaking if you rename the folder

```
Client code uses the package path when importing the package. By convention, the last element of the package path is the package name
```
https://blog.golang.org/package-names

**Pull request merged**